Release notes for March 17, 2017

marcotuts opened the pull request:
The following pull request includes the read the docs changes necessary for the week ending March 17, 2017 

### Release Notes Page

The confluence page for this week's release notes can be found here: [Release Notes: Week Ending March 17, 2017](https://openedx.atlassian.net/wiki/pages/viewpage.action?pageId=158212641)

### Date Needed (optional)

EOD Monday March 20, 2017
### Reviewers
Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Doc team review: @catong

FYI: @srpearce

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version 

- [ ] http://draft-release-notes.readthedocs.io/en/latest/2017/2017-03-17.html

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Squash commits

catong commented on en_us/release_notes/source/2017/lms/lms_2017-03-17.rst:
```diff
@@ -0,0 +1,8 @@
+Course teams can now target emails specifically to track (i.e., audit and
```
"to track" -> "to learners who are in a particular enrollment track" 
"(i.e. audit and verified)" -> "for example, audit or verified."
(remove parentheses)

catong commented on en_us/release_notes/source/2017/lms/lms_2017-03-17.rst:
```diff
@@ -0,0 +1,8 @@
+verified). (:jira:`TNL-6703`)
+
+To improve accessibility, we have added a notification to our basic problems
+when 'Show Answer' is pressed. The 'Show Answer' button will now display a
```
"will now display" -> "now displays"
"available inline" -> "available in the body of the problem"

catong commented on en_us/release_notes/source/2017/website/website_2016-09-26.rst:
```diff
@@ -0,0 +1,4 @@
+Allow a learner to enroll in any available course run from a course and
+program about page (via a dialog box that features all available course run
+dates). (:JIRA:`ECOM-7262`)
+
```
@marcotuts This erroneous update to this file (2016-09-26) should be discarded.

catong commented on en_us/release_notes/source/2017/website/website_2017-03-17.rst:
```diff
@@ -0,0 +1,3 @@
+Allow a learner to enroll in any available course run from a course and
+program about page (via a dialog box that features all available course run
+dates). (:JIRA:`ECOM-7262`)
```
@marcotuts @MatthewPiatetsky I thought this was still in progress? Not sure this was already in Prod last week

marcotuts replied:
Hmm... the release note field may have been seen set on a ticket that was merged but that doesn't reflect feature completion. I will push to next week and exclude from this week for now, and we can keep punting it to future weeks using labels until it lands. thanks for catching this @catong

marcotuts commented:
@catong  I've addressed the feedback and this is I think ready for a final review. building locally now.

catong commented:
@marcotuts LGTM for merging after commits squashed.

marcotuts commented:
awesome! squashed commits, merged PR, updated gh-pages branch. I think this just needs a build button push on the docs site. Will update / create portal announcements now.
